src/azul/plugins/metadata/hca/indexer/aggregate.py

```diff
@@ -124,13 +124,14 @@ def _transform_entity(self, entity: JSON) -> JSON:
         return {
             **entity,
             'total_estimated_cells': (entity['document_id'], entity['total_estimated_cells']),
+            'actual_total_estimated_cells': (entity['document_id'], entity['actual_total_estimated_cells']),
```

Comment:
I find the prefix "actual" confusing because it implies that `actual_total_estimated_cells` is the real or correct value and that `estimated_cell_count` is therefore somehow fake or inaccurate. Since `estimated_cell_count` is the field we actually want users to look at, I think it would be better to replace "actual" with a different qualifier. Some ideas for what this qualifier could be include "literal", "pass_through", "uncorrected", "unadjusted", "standalone", etc. I'd recommend discussing with Hannes.

Reply:
Easy enough to change, I'll go with `total_estimated_cells_unadjusted` for now, and discuss with Hannes in next PL session.